src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
+enum Shapes {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
+
 export interface Figure {
+  shape: string;
+  color: string;
 
+  getArea: () => number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: string = Shapes.triangle;
```

Comment:
you can omit public keyword, shape would be public by default

```suggestion
  shape: string = Shapes.triangle;
```

Reply:
++